src/librbd/io/CopyupRequest.cc

```diff
@@ -18,7 +18,7 @@
 #include "librbd/io/ObjectRequest.h"
 #include "librbd/io/ReadResult.h"
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```

Comment:
Nit: just drop the include since it appears not to be used anymore

Reply:
Fixed